src/Engine/Spells/Spells.cpp

```diff
@@ -444,8 +445,9 @@ void SpellStats::Initialize() {
 
 void EventCastSpell(int uSpellID, PLAYER_SKILL_MASTERY skillMastery, PLAYER_SKILL_LEVEL skillLevel, int fromx,
                     int fromy, int fromz, int tox, int toy, int toz) {
-    Assert(skillMastery >= PLAYER_SKILL_MASTERY_NOVICE && skillMastery <= PLAYER_SKILL_MASTERY_GRANDMASTER,
-           "Invalid mastery level");
+    // TODO(pskelton): why was this here?
+    // Assert(skillMastery >= PLAYER_SKILL_MASTERY_NOVICE && skillMastery <= PLAYER_SKILL_MASTERY_GRANDMASTER,
+    //       "Invalid mastery level");
```

Comment:
This function expect defined spell mastery in range from novice to grandmaster, please leave this assert in place it catched one bug already.

Reply:
Commented out when it kept asserting whilst I was testing something else. Will revert.

Reply:
there's still a lot of integers passing around instead of the enum values.
but it was much worse before, these asserts were basically to catch decompilation or sloppy code rewrites if decompilation was a mess